lib/routes.js

```diff
@@ -3,7 +3,7 @@ const axios = require('axios');
 module.exports = function (self, options) {
   self.addRoutes = function() {
     self.route('post', 'page-scan-modal', async (req, res) => {
-      if (!self.apos.permissions.can(req, 'admin')) {
+      if (!(self.apos.permissions.can(req, 'admin') || self.apos.permissions.can(req, 'admin-seo'))) {
```

Comment:
A user with the admin permission automatically has all other permissions, so you can check just for the admin-seo permission.

Reply:
Done